tests/functional/conftest.py

```diff
@@ -20,6 +20,17 @@ def fixture_dir(test_dir):
 def reset_gitlab(gl: gitlab.Gitlab) -> None:
     """Delete resources (such as projects, groups, users) that shouldn't
     exist."""
+    if is_gitlab_ee(gl):
+        logging.info("GitLab EE detected")
+        # NOTE(jlvillal): By default in GitLab EE it will wait 7 days before
+        # deleting a group. Change it to 0 days.
+        settings = gl.settings.get()
+        logging.info(f"deletion_adjourned_period: {settings.deletion_adjourned_period}")
```

Comment:
To me this looks like a debug entry, maybe? We have the info below for when it's actionable when it finds 0.

Reply:
Done